docs/quickstart.md

```diff
@@ -2,6 +2,7 @@
 
 - [Installation Quickstart](#installation-quickstart)
   - [Download the Automation Code](#download-the-automation-code)
+  - [Setup Terraform Environment](#setup-Terraform-environment)
```

Comment:
This section should be in https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/docs/automation_host_prereqs.md#automation-host-setup where we ask user to setup Terraform binary.

Reply:
done.